pinot-servicemanager/schemas/spanEventView-schemaFile.json

```diff
@@ -126,6 +126,11 @@
       "name": "space_ids",
       "dataType": "STRING",
       "singleValueField": false
+    },
+    {
+      "name": "total_span_count",
```

Comment:
How will this work given that this will be a column each span record?

Reply:
It is basically the count of spans in the tree rooted at this particular span. 
Refer https://github.com/hypertrace/hypertrace-ingester/pull/130